src/app/components/Tokens/TokenList.tsx

```diff
@@ -0,0 +1,82 @@
+import { useTranslation } from 'react-i18next'
+import { EvmToken } from '../../../oasis-indexer/api'
+import { Table, TableCellAlign, TableColProps } from '../../components/Table'
+import { TablePaginationProps } from '../Table/TablePagination'
+import { AccountLink } from '../Account/AccountLink'
+import { TokenLink } from './TokenLink'
+import { CopyToClipboard } from '../CopyToClipboard'
+
+type TokensProps = {
+  tokens?: EvmToken[]
+  isLoading: boolean
+  limit: number
+  verbose?: boolean
+  pagination: false | TablePaginationProps
+}
+
+export const TokenList = (props: TokensProps) => {
+  const { isLoading, tokens, pagination, limit } = props
+  const { t } = useTranslation()
+  const tableColumns: TableColProps[] = [
+    { content: '' },
+    { content: t('common.name') },
+    { content: t('common.smartContract') },
+    {
+      content: t('tokens.holdersCount'),
+      align: TableCellAlign.Right,
+    },
+    { content: t('tokens.supply'), align: TableCellAlign.Right },
+    { content: t('common.ticker'), align: TableCellAlign.Right },
+  ]
+
+  const tableRows = tokens?.map((token, index) => {
+    return {
+      key: token.contract_addr,
+      data: [
+        {
+          content: (index + 1).toLocaleString(),
```

Comment:
You need to take an offset into account.

Reply:
Right. Fixed.